AEPAudience/Tests/AudienceTests.swift

```diff
@@ -468,28 +482,26 @@ class AudienceTests: XCTestCase {
 
     func testHandleLifecycleResponse_LifecycleResponseHasEmptyData() {
         // setup
-        // dispatch a configuration response event containing aam timeout, privacy status opted in, aam server, and aam forwarding status equal to false
-        let configData = dispatchConfigurationEventForTesting(aamServer: "testServer.com", aamForwardingStatus: false, privacyStatus: .optedIn, aamTimeout: 10)
-        // create lifecycle response content
-        let lifecycleContextData:[String: Any] = [String: Any]()
+        mockConfiguration(aamServer: "testServer.com",
+                          aamForwardingStatus: false,
+                          privacyStatus: .optedIn,
+                          aamTimeout: 10)
         // create the lifecycle event and simulate having the configuration data in shared state
-        let lifecycleEvent = Event(name: "Test Lifecycle response", type: EventType.lifecycle, source: EventSource.responseContent, data: lifecycleContextData)
-        mockRuntime.simulateSharedState(extensionName: AudienceConstants.SharedStateKeys.CONFIGURATION, event: lifecycleEvent, data: (configData, .set))
+        let lifecycleEvent = createLifecycleResponseEvent(withContextDataData: [:])
         let _ = audience.readyForEvent(lifecycleEvent)
 
         // test
         mockRuntime.simulateComingEvent(event: lifecycleEvent)
 
         // verify
-        XCTAssertEqual(0, audience.state?.hitQueue.count())
+        XCTAssertEqual(1, audience.state?.hitQueue.count())
```

Comment:
this assertion changed because signals with empty data are queued and the initial lifecycle context data format was incorrect. I am open for suggestions to ignore events with empty lifecycle data.